timeseries/src/autogluon/timeseries/models/chronos/model.py

```diff
@@ -218,12 +245,12 @@ def get_minimum_resources(self, is_gpu_available: bool = False) -> Dict[str, Uni
             minimum_resources["num_gpus"] = self.min_num_gpus
         return minimum_resources
 
-    def load_model_pipeline(self, context_length: Optional[int] = None):
+    def load_model_pipeline(self, context_length: Optional[int] = None, is_training: bool = False):
         from .pipeline import OptimizedChronosPipeline
 
         gpu_available = self._is_gpu_available()
 
-        if not gpu_available and self.min_num_gpus > 0:
+        if not gpu_available and (self.min_num_gpus > 0 or is_training):
```

Comment:
Do we require GPU for fine-tuning because HF will raise an error if no GPU provided, or will the fine-tuning just be extremely slow?

If it's the latter, is fine-tuning on CPU also slow for the patch-based model?

Reply:
It will be slow. Should we allow CPU fine-tuning? (I think for Bolt CPU fine-tuning should be possible at a reasonable speed)

Reply:
I think CPU fine-tuning is reasonable for Bolt, so I would allow this.

Reply:
Fixed.